database/migrations/2018_02_22_160436_add_remote_user_settings.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddRemoteUserSettings extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('settings', function (Blueprint $table) {
+            $table->boolean('login_remote_user_enabled')->nullable()->default(0);
```

Comment:
This should maybe not be nullable, but instead just default to 0? A null condition here for users who have never set (either allowing or denying) seems like it could have unpredictable results.